src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -59,9 +63,93 @@ constexpr int8_t kDefaultDeadBand                 = 25; // 2.5C is the default
 #define FEATURE_MAP_SCH 0x08
 #define FEATURE_MAP_SB 0x10
 #define FEATURE_MAP_AUTO 0x20
+#define FEATURE_MAP_LTNE 0x40
 
 #define FEATURE_MAP_DEFAULT FEATURE_MAP_HEAT | FEATURE_MAP_COOL | FEATURE_MAP_AUTO
 
+class ThermostatAttrAccess : public AttributeAccessInterface
+{
+public:
+    ThermostatAttrAccess() : AttributeAccessInterface(Optional<EndpointId>::Missing(), Thermostat::Id) {}
+
+    CHIP_ERROR Read(const ConcreteReadAttributePath & aPath, AttributeValueEncoder & aEncoder) override;
+    CHIP_ERROR Write(const ConcreteDataAttributePath & aPath, AttributeValueDecoder & aDecoder) override;
+};
+
+ThermostatAttrAccess gThermostatAttrAccess;
+
+CHIP_ERROR ThermostatAttrAccess::Read(const ConcreteReadAttributePath & aPath, AttributeValueEncoder & aEncoder)
+{
+    VerifyOrDie(aPath.mClusterId == Thermostat::Id);
+
+    bool LTNESupported = false;
+    uint32_t OurFeatureMap;
+
+    if (FeatureMap::Get(aPath.mEndpointId, &OurFeatureMap) == EMBER_ZCL_STATUS_SUCCESS &&
+        OurFeatureMap & 1 << 6) // Bit 6 is LTNE supported
```

Comment:
I know this is how the rest of the feature checks are done in this file, but ideally this would be more like:
```
  OurFeatureMap & to_underlying(Feature::kLocalTemperatureNotExposed)
```
after https://github.com/project-chip/connectedhomeip/pull/26673 merges or:
```
  OurFeatureMap & to_underlying(ThermostatFeature::kLocalTemperatureNotExposed)
```
before that.  Followup for fixing this up is OK, especially if it fixes up all the feature checks in this cluster, and then can happen after #26673.